puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-discovery-image.groovy

```diff
@@ -0,0 +1,67 @@
+pipeline {
+    agent { label 'admin && sshkey' }
+
+    options {
+        timestamps()
+        timeout(time: 3, unit: 'HOURS')
+        disableConcurrentBuilds()
+        ansiColor('xterm')
+    }
+
+    stages {
+        stage('Run FDI build') {
+            steps {
+                deleteDir()
+                dir('build') {
+                    script {
+                        job_parameters = [
+                            proxy_repository: env.proxy_repository,
+                            branch: env.branch,
+                        ]
+                        job_extra_vars = [
+                            jenkins_download_artifacts: 'true',
+                            jenkins_artifacts_directory: '../result/',
+                        ]
+                        runCicoJob("foreman-discovery-image-build", job_parameters, job_extra_vars)
+                    }
+                }
+            }
+        }
+
+        stage('Prepare FDI upload') {
+            steps {
+                sh """
+                pushd result
+                ln -snf fdi*tar fdi-image-latest.tar
+                md5sum fdi*tar fdi*iso > MD5SUMS
+                popd
+                """
+            }
+        }
+
+        stage('Publish FDI build') {
+            steps {
+                script {
+                    base_dir = "/var/www/vhosts/downloads/htdocs/discovery"
+                    destination_user = 'root'
+                    destination_server = 'web02.theforeman.org'
+                    destination_dir = "${base_dir}/${output_dir}"
+
+                    sshagent(['deploy-downloads']) {
+                        // publish on web
+                        sh "rsync -av --delete result/ ${destination_user}@${destination_server}:${destination_dir}/"
```

Comment:
hmm, `result` currently contains more artifacts than we need to ship. might need to clean that up.

Reply:
fixed